Tests/PackageCollectionsTests/PackageCollectionsTests.swift

```diff
@@ -1224,6 +1328,10 @@ final class PackageCollectionsTests: XCTestCase {
         try XCTSkipIf(true)
         #endif
 
+        if !PackageCollections.isSupportedPlatform {
+            try XCTSkipIf(true)
```

Comment:
`try XCTSkipIf(!PackageCollections.isSupportedPlatform)`

Reply:
I *think* there is a way in XCTest to apply such rules to all tests in the class, which may be easier. @neonichu do you remember how?

Reply:
Not aware of anything tbh